fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/RecordCursorTest.java

```diff
@@ -1268,4 +1278,58 @@ void asStreamTest() {
 
     }
 
+    @RepeatedTest(100)
```

Comment:
Opening the draft PR with this repetition, but I'm going to remove it before merging, as it seems excessive.